apps/admin-x-settings/src/components/settings/general/users/RoleSelector.tsx

```
@@ -56,6 +56,43 @@ const RoleSelector: React.FC<{ user: User; setUserData: (user: User) => void; }>
         });
     }
 
+    let optionsArray = [
```

Biome (1.9.4)

[error] Shouldn't redeclare 'optionsArray'. Consider to delete it or rename it.

'optionsArray' is defined here:


(lint/suspicious/noRedeclare)